permissions/plugin-upload-pgyer-plugin.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "upload-pgyer"
+paths:
+- "ren/helloworld/upload-pgyer"
+developers:
+- "fan yang"
```

Comment:
Not a user name in Artifactory, and I don't think we support spaces. Please see this repository's README on what to enter here.

Reply:
updated